doc/configuration_options.rst

```diff
@@ -472,6 +472,8 @@ The following options are valid in version ``1`` (beside the generic options):
     implicit zero value from the y-axis.
   * ``y_axis_minimum``: Minimum y-axis value.
   * ``y_axis_maximum``: Maximum y-axis value.
+  * ``num_build``: Number of builds back to show on this plot.
+  An empty string means all builds. Must not be "0".
```

Comment:
Needs to be indented two spaces more. Second sentence still needs to be wrapped. Document the default value.

Reply:
The second part of my comment still applies.